cadasta/api/project.py

```diff
@@ -24,8 +24,7 @@ class Project(NetworkMixin):
     api_url = 'api/v1/projects/'
 
     def __init__(self, on_finished=None):
-        """
-        Constructor.
+        """ Constructor.
```

Comment:
Minor things, we usually don't add space after `"""`. Please read https://github.com/inasafe/inasafe/wiki/Coding-Standards#doc-strings-and-comments